src/core/shape/plane.js

```diff
@@ -39,6 +37,16 @@ class Plane {
         }
     }
 
+    /**
+     * The d term for a scalar equation of the form ax + by + cz + d = 0, where a, b and c
+     * are the x, y and z components of the plane's normal.
+     *
+     * @type {number}
+     */
+    get d() {
```

Comment:
shouldn't we fix plane before making it public api? why on earth it stores a normal and point is beyond me!

Reply:
yep, that makes sense and done. Larger PR, but hey.